gradle.properties

```diff
@@ -7,5 +7,3 @@ loom.platform=forge
 # See BuildFlags.java for usages of these values.
 # Build with Pronouns in PV by default
 # neu.buildflags.pronouns=true
-# Build with Oneconfig support
-# neu.buildflags.oneconfig=true
```

Comment:
no

Reply:
yeah but its commented out
i dont think neu comes with oneconfig by default which is one of the things the flag does